analysis/model/src/test/kotlin/de/maibornwolff/codecharta/serialization/OutputFileHandlerTest.kt

```diff
@@ -0,0 +1,49 @@
+package de.maibornwolff.codecharta.serialization
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class OutputFileHandlerTest {
```

Comment:
Please add more strings to be tested like "my.long.map.name.cc.json" or ".hiddeenfile" and so on. We should forget to test all the edge cases.

Reply:
how about cases like "my.long.map.name". when using substringBeforeLast, we would get : "my.long.map.cc.json", which would be incorrect. shall we use some function that checks if the last substring is any file-ending or shall we only check for .json or .cc.json since we would not expect any other file ending

Reply:
Your suggestion sounds good. Let's expect json and cc.json only.